src/Entity/MetadataDisplayEntity.php

```diff
@@ -441,27 +446,47 @@ public function getTwigVariablesUsed() {
    * @return array
    *   A list of used $variables by this template.
    */
-  private function getTwigVariableNames(ModuleNode|Node|BodyNode $nodes): array {
+  private function getTwigVariableNames(ModuleNode|Node|BodyNode $nodes, $all_variables): array {
```

Comment:
@aksm what is $all_variables here? Type? default? I don't see an explanation on the PHP DOC

Reply:
@DiegoPino Sorry, forgot about the PHP DOC. Updated via 4fb2803ee2541a7410c2b9808e40ffee28cb91e0 and 790b4ad4d4602ce844db28f4009b07718978f834. Does that make sense?